data/provider/src/main/java/tech/pegasys/teku/api/ValidatorDataProvider.java

```diff
@@ -371,7 +371,9 @@ public void subscribeToSyncCommitteeSubnets(
 
   public void prepareBeaconProposer(
       Collection<BeaconPreparableProposer> beaconPreparableProposers) {
-    List<tech.pegasys.teku.spec.datastructures.operations.versions.merge.BeaconPreparableProposer>
+    List<
+            tech.pegasys.teku.spec.datastructures.operations.versions.bellatrix
+                .BeaconPreparableProposer>
```

Comment:
wow hopefully that wasn't spotless that did that...

Reply:
It was.